e2e/libs/workload/pod.py

```diff
@@ -31,15 +31,35 @@ def new_pod_manifest(image="", command=[], args=[],
         },
         'spec': {
             'nodeName': node_name,
+            'restartPolicy': 'Never',
             'containers': [{
                 'image': image,
                 'imagePullPolicy': 'IfNotPresent',
+                'securityContext': {
+                    'privileged': True
+                },
                 'name': 'run',
                 'command': command,
                 'args': args,
-                'volumeMounts': [],
+                'volumeMounts': [{
+                    'name': 'bus',
+                    'mountPath': '/var/run'
+                }, {
+                    'name': 'rancher',
+                    'mountPath': '/var/lib/rancher'
+                }]
             }],
-            'volumes': []
+            'volumes': [{
+                'name': 'bus',
+                'hostPath': {
+                    'path': '/var/run'
+                }
+            }, {
+                'name': 'rancher',
+                'hostPath': {
+                    'path': '/var/lib/rancher'
+                }
+            }]
```

Comment:
Could we have a new function `new_pod_manifest_for_node_exec` that uses `new_pod_manifest` for the base manifest and then append the volumes fields?

Reply:
Can be refined later after sort out what are the possible required configurations for different situations.